internal/authorization/rbac/rbac.go

```diff
@@ -153,7 +155,7 @@ func verbMatches(rule *rbacv1.PolicyRule, requestedVerb string) bool {
 		if ruleVerb == rbacv1.VerbAll {
 			return true
 		}
-		if strings.ToLower(ruleVerb) == strings.ToLower(requestedVerb) {
+		if strings.EqualFold(ruleVerb, requestedVerb) {
```

Comment:
Is there any security ramifications of using EqualFold? This does more than just case-insensitive comparisons? cc @mhrabovcin